File: SeriousSez/src/app/fridges/models/Fridge-grocery.ts
import { Image } from "../../recipe/models/image.interface";

export interface FridgeGrocery {
    id: string;
    name: string;
    description: string;
    amount: number;
    amountType: string;
    purchaseDate: string;
    expirationDate: string;
    image: Image | null;
    fridgeId: string;
}

File: SeriousSez.Domain/Entities/Fridge/FridgeGrocery.cs
using SeriousSez.Domain.Entities.Recipe;
using System;

namespace SeriousSez.Domain.Entities.Fridge
{
    public class FridgeGrocery : BaseIngredient
    {
        public decimal Amount { get; set; }
        public string AmountType { get; set; }
        public DateTime PurchaseDate { get; set; }
        public DateTime ExpirationDate { get; set; }
        public Image Image { get; set; }
        public Guid FridgeId { get; set; }
    }
}
